tutorials/graph-hello-world/graph-hello-world.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Hello Graph! Write Your First SAP Graph Application
+description: Get a brief introduction to SAP Graph, including the SAP Graph Navigator and programming basics.
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product>sap-business-technology-platform]
+primary_tag: software-product>sap-integration-suite
+---
+
```

Comment:
Further down you are mentioning NodeJs. This belongs and anything the user needs to do before he can start with the tutorial belongs in the "Prerequisites" section.

Reply:
Moved to prereqs section.

Reply:
This section hast to come before the "You will learn" section.